docs/components/modeler/web-modeler/process-applications.md

```diff
@@ -0,0 +1,126 @@
+This reduces the risk of having a broken deployment at runtime and makes it more convenient to deploy related files.
+
+We advise using a process application if there is one main BPMN process that represents your end-to-end use case and
+additional files the main process depends on – like called supporting processes, DMN decisions, or forms.
+
+## Create a process application
+
+Create a new process application by clicking the **Create process application** button in an empty project or via
+**Create new** > **Process application** on the project or folder page.
+
+<p><img src={EmptyProjectImg} alt="Create a process application from an empty project" /></p>
+
+You will be asked to provide a name for the new process application.
+
+<p><img src={CreateProcessApplicationImg} style={{width: 500}} alt="Create a new process application" /></p>
+
+### Main process
+
+The new process application will contain an empty BPMN diagram with the same name as the process application itself,
+labelled as the main process.
+You can rename the main process diagram any time if you wish.
+
+:::note
+A process application must always have a main process.
+That is why the main process diagram cannot be deleted or moved out of the process application.
+:::
+
+### Add files to a process application
+
+Add more files to the process application via the **Create new** dropdown on the process application page.
+You can also drag and drop files from your local computer there or move existing files from a different location in
+Web Modeler to the process application.
+
+To make it easily discoverable, the main process will always be displayed at the top of the file list, also when you change its sort order.
+
+<p><img src={FileListImg} alt="Process application file list" /></p>
+
+:::note
+Within the same process application, process IDs, decision IDs and form IDs must be unique across all files.
+This is to avoid ambiguity and conflicts when you link resources and deploy the process application later on.
+:::
+
+## Deploy and run a process application
+
+### Deploy a process application
+
+Use the **Deploy** button on the process application page to deploy the process application to a Zeebe cluster.
+
+<p><img src={DeployProcessApplicationImg} alt="Deploy a process application" /></p>
+
+All resources contained in the process application (i.e. the main process plus all other BPMN diagrams, DMN diagrams and
+forms) will be deployed together in a single bundle.
+If any of the resources is invalid and cannot be deployed, the whole deployment will [fail](#deployment-errors) and the
+cluster state will remain unchanged.
+This ensures that a process application cannot be deployed incompletely or in an inconsistent state, which makes it safer to use.
+
+### Run a process application
+
+Use the **Run** button on the process application page to start a new instance of the process application's [main process](#main-process).
```

Comment:
When I paired with Valentino yesterday, the **Run** button was on both the process application page and when viewing the main process in the editor page. Has that changed?

Reply:
@HanselIdes The **Run** button is actually displayed on the process application page and on the editor page for _every_ diagram in the process application (I just noticed that is displayed for both BPMN and DMN diagrams).